src/annotator/components/NotebookModal.js

```diff
@@ -48,6 +48,10 @@ function NotebookIframe({ config, groupId }) {
  * @param {NotebookModalProps} props
  */
 export default function NotebookModal({ eventBus, config }) {
+  // Temporal solution: while there is no mechanism to sync new annotations in
```

Comment:
Is there a ticket to track this sync improvement? It might be good to add a link in the comments here if there is one.

Reply:
I added a ticket.